### ISSUE TITLE
Reputation updates

### ISSUE THREAD

**Issue by elenadimitrova** [JoinColony/colonyNetwork#39]
### 6.2.1 Reputation change from contributing to a task
For all participants, reputation updates occur and payouts are made available only after the objection window (described in Section 4.1) has closed and all disputes (described in Section 9) have been resolved at the end of the task. The reputation updates and payouts are based on the final state of the task and the difference, if any, between the initial gradings and final state of the task.

**Comment by elenadimitrova** [comment]
Reputation updates done in #130 #134 and #158 . Objections and disputes not part of the MVP scope.

**Comment by kronosapiens** [comment]
No longer relevant.
